### PR TITLE
Add organisation documentation

### PR DESCRIPTION
**Pull request by sleepypioneer**
This is a rather large PR, and I am inclined to already merge this and then work on top of it for edits as the information is already useful and would consider this to always be an evolving piece of documentation.

The idea was to have a starting point to help make content more discoverable, almost in the style of a tell-your-own-story. This could later be made into a wiki here on github or hosted in some sort of web form.

### CODE REVIEW

**Comment by marimeireles** [comment]
Ah wow!
You were already adding it to the website, I thought you meant the github wiki!
Great work Jessica, thank you! ✨
I'm available to review it, just ping me :D

**Comment by sleepypioneer** [comment]
@marimeireles this isn't actually the website, I would agree with adding something there! Maybe even all of this in some form if possible but I think markdown pages is ok for now.

**Comment by marimeireles** [comment]
Oh, is just MDs with info? 
Yeah, it'd be very easy to just move them to the website... The hardest part is done. =)

**Comment by sleepypioneer** [comment]
This is still incomplete but I feel it might be worth merging it already and adding on top of it, what do folks think?

File: organizing/sub_pages/supporting_pyladies.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+*"For a long time I felt like I did not belong in tech. On the way, I met many people who felt as I did. This motivated me to try my best to make others feel included in the PyLadies community. Through building a community, I have found my way to belong as well."*
+***- L***
+
+
+*"It is because of this group, in chapters in Rio de Janeiro, Mexico City and especially Berlin, that I was able to change careers and meet my closest care network. Especially in Berlin, a city with more than 30% foreigners far from their family network, PyLadies makes a bit of a difference in life."*
+***- P***
+
+
+*"I met some of the most supportive and inspiring organizers at PyLadies. It’s a great place to share knowledge and learn. To get motivated, and inspired, and pass on sponsorship opportunities as you get ahead in you career - pay it forward. I can’t think of a better network with worthier more hardworking, giving and sincere folks. I truly cherish these connections and experiences. So grateful."*
+***- N***
```
Comment (noatamir) [inline organizing/sub_pages/supporting_pyladies.md]:
```suggestion
***- Noa Tamir***
```

**Comment by noatamir** [comment]
I read through all the new content, and it looks really great @sleepypioneer!
I haven't checked for what's dropped. I agree that it's worthwhile adding this as is and keep adding or amending on smaller PRs later.
Is there anything specific you want more feedback on?

**Comment by sleepypioneer** [comment]
As discussed merging this now and will make some follow up PRs.

**Pull request merged**